superset-frontend/src/views/CRUD/welcome/Welcome.tsx

```diff
@@ -279,7 +279,7 @@ function Welcome({ user, addDangerToast }: WelcomeProps) {
   return (
     <WelcomeContainer>
       <WelcomeNav>
-        <h1 className="welcome-header">Home</h1>
+        <h1 className="welcome-header">{t('Home')}</h1>
```

Comment:
Thanks for catching this!

Reply:
correction is in this PR : https://github.com/apache/superset/pull/20078